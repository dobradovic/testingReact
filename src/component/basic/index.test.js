import React from "react";
import { shallow } from "enzyme";
import Basic from "./index";
import { findByTestAttr } from "./../../../Utils";

describe("testing basic component", () => {
  it("renders correctly enzyme", () => {
    const wrapper = shallow(<Basic />);
    expect(wrapper).toMatchSnapshot();
  });
});
