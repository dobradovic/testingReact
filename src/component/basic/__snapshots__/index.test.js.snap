// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing basic component renders correctly enzyme 1`] = `ShallowWrapper {}`;
